pyccl/baryons/vandaalen19.py

```diff
@@ -0,0 +1,94 @@
+__all__ = ("vanDaalen19",)
+
+import numpy as np
+
+from .. import Pk2D
+from . import Baryons
+
+
+class vanDaalen19(Baryons):
+    """The "van Daalen+ 2019" model boost factor for baryons.
+
+    .. note:: First presented in van Daalen et al., arXiv:1906.00968.
+
+              The boost factor is applied multiplicatively so that
+              :math:`P_{\\rm bar.}(k, a) = P_{\\rm DMO}(k, a)\\,
+              f_{\\rm vD19}(k, a)`.
+
+              Notice the model has only been tested at z=0 and is valid for
+              :math:`k\\leq 1 \\,h/{\\rm Mpc}.
+
+    Args:
+        fbar500c (:obj:`float`): the fraction of baryons in a halo within 
```

Comment:
Do you think we could allow for the 200c mass definition from their table 3 too? I just remember how annoying it is to have to translate between them if you can't choose to use 500c. E.g. you could rename this to simply `fbar` and then add another argument of the form `mass_def='500c'`, but which users could change to `'200c'`

Reply:
Done